HelpAssistHer/help-assist-her
# Services group

## Description
This PR adds the full list of services and formats it correctly. In order to make the code reusable, scalable, and give us the ability to format it according to the designs, I created several components:

Services > Services Group > Service Fields > Services Checkbox

where:
Services = All of the Service related code for the form
Services Group = each group of services with appropriate heading
Service Fields = the list of the fields in each group
Services Checkbox = the formatted checkbox for each service

## Test Plan
1. Load the app
1. Go to the services section and make sure it matches the designs. There should be 6 groups of services.
1. Click Verify Next Resource until you load a resource that has services. Make sure the correct services are automatically selected.
1. Click a couple services for a resource and save. Make sure they are successfully saved.
